README.md

```diff
@@ -240,12 +253,166 @@ sl_reanalysis = xr.open_zarr(
+geopotential                                     | z	      | m^2 s^-2  | https://apps.ecmwf.int/codes/grib/param-dbid=129
+high_vegetation_cover                            | cvh        | (0 - 1)   | https://codes.ecmwf.int/grib/param-db/?id=28
+lake_cover                                       | cl         | (0 - 1)   | https://codes.ecmwf.int/grib/param-db/?id=26
+lake_depth                                       | dl         | m         | https://codes.ecmwf.int/grib/param-db/?id=228007
+land_sea_mask                                    | lsm        | (0 - 1)   | https://apps.ecmwf.int/codes/grib/param-db?id=172
+low_vegetation_cover                             | cvl        | (0 - 1)   | https://apps.ecmwf.int/codes/grib/param-db?id=27
+mean_sea_level_pressure                          | msl        | Pa        | https://codes.ecmwf.int/grib/param-db/?id=151
+sea_surface_temperature                          | sst        | K         | https://codes.ecmwf.int/grib/param-db/?id=134
+slope_of_sub_gridscale_orography                 | slor       | ~         | https://codes.ecmwf.int/grib/param-db/?id=163
+soil_type                                        | slt        | ~         | https://codes.ecmwf.int/grib/param-db/?id=43
+specific_humidity                                | q          | kg kg**-1 | https://codes.ecmwf.int/grib/param-db/?id=133
+standard_deviation_of_filtered_subgrid_orography | sdfor      | m         | https://codes.ecmwf.int/grib/param-db/?id=74
+standard_deviation_of_orography                  | sdor       | ~         | https://codes.ecmwf.int/grib/param-db/?id=160
+surface_pressure                                 | sp         | Pa        | https://codes.ecmwf.int/grib/param-db/?id=134
+toa_incident_solar_radiation                     | tisr       | J m**-2   | https://codes.ecmwf.int/grib/param-db/?id=212
+total_cloud_cover                                | tcc        | (0 - 1)   | https://codes.ecmwf.int/grib/param-db/?id=164
+total_column_water_vapour                        | tcwv       | kg m**-2  | https://codes.ecmwf.int/grib/param-db/?id=137
+type_of_high_vegetation                          | tvh        | ~         | https://codes.ecmwf.int/grib/param-db/?id=74
+type_of_low_vegetation                           | tvl        | ~         | https://codes.ecmwf.int/grib/param-db/?id=29
+u_component_of_wind                              | u          | m s**-1   | https://codes.ecmwf.int/grib/param-db/?id=131
+v_component_of_wind                              | v          | m s**-1   | https://codes.ecmwf.int/grib/param-db/?id=132
+vertical_velocity                                | w          | Pa s**-1  | https://codes.ecmwf.int/grib/param-db/?id=135
+```python
+import xarray as xr
+
+ar_full_37_6h = xr.open_zarr(
+    'gs://gcp-public-data-arco-era5/ar/1959-2022-full_37-6h-0p25deg-chunk-1.zarr-v2/', 
+)
+```
+### 1959-2022-wb13-6h-0p25deg-chunk-1.zarr-v2/
```

Comment:
Let's get rid of this one, too.

Reply:
will remove this once `AR-v3` completed.